calico_node/startup/startup.go

```diff
@@ -526,10 +544,16 @@ func ipv6Supported() bool {
 
 // createIPPool creates an IP pool using the specified CIDR.  This
 // method is a no-op if the pool already exists.
-func createIPPool(client *client.Client, cidr *net.IPNet, ipipModeName string) {
+func createIPPool(client *client.Client, cidr *net.IPNet, ipipModeName string, isNATOutgoingDefined bool, isNATOutgoingEnabled bool) {
 	version := cidr.Version()
 	ipipMode := ipip.Mode(ipipModeName)
 
+
+	//Workaround for NatOutgoing where IPv6 default is false
+	if(version == 6 && isNATOutgoingDefined == false) {
```

Comment:
I think this should be placed in configureIPPools, there is already an IPv6 section there and then it won't be necessary to pass isNATOutgoingDefined into createIPPool.